Sources/ApodiniAuthorization/AuthorizedProperty.swift

```diff
@@ -8,16 +8,16 @@
 
 import Apodini
 
-/// The ``Authorized`` `DynamicProperty` can be used to access the ``Authenticatable``
-/// instance created through a authorization Metadata.
+/// The ``Authorized`` property wrapper can be used to access the ``Authenticatable``
+/// instance created through an authorization Metadata.
 ///
 /// Given the example of an `ExampleUser` ``Authenticatable``, the following code may be used to
 /// access the authenticated and authorized instance.
 /// ```swift
 /// struct ExampleHandler: Handler {
-///     var authorizedUser = Authorized<ExampleUser>()
+///     @Authorized(ExampleUser.self) var authorizedUser
```

Comment:
Maybe even `@Authorized var authorizedUser: ExampleUser` looks more natural? As far as I can see the default value in initializer would enables this?

Reply:
Isn't that the whole problem that `wrappedValue` can't be throwing? Afaik - and maybe its only me not knowing about the internals of a property wrapper well enough - but the type of the property on which the property wrapper is used on must match the type of the `wrappedValue`, right?
As we have the type `AuthorizedAuthenticatable<Element>` of the `wrappedValue` the propertyWrapper forces the type of the property to `AuthorizedAuthenticatable<Element>`.

Like the following two would be equivalent:
```swift
@Authorized(ExampleUser.self) var authorizedUser
@Authorized(ExampleUser.self) var authorizedUser: AuthorizedAuthenticatable<ExampleUser>

// and probably that works also due to the default init param
@Authorized var authorizedUser: AuthorizedAuthenticatable<ExampleUser>
```

But I'm not sure how your approach would work? Can the type of the `init(wrappedValue:)` be different to the type of the `wrappedValue`?